src/databricks/labs/ucx/assessment/queries/all_tables.sql

```diff
@@ -0,0 +1,14 @@
+-- viz type=table, name=Table Types, columns=database,name,type,table_format,table_view,storage,is_delta,location
+-- widget title=Table Types, col=0, row=0, size_x=6, size_y=6
+SELECT
+  `database`,
+  name,
+  object_type AS type,
+  upper(table_format) AS format,
```

Comment:
```suggestion
  UPPER(table_format) AS format,
```

nit. But we really should change the tables.scala to do the upper formatting

Reply:
Can be done in a follow up PR.